src/__tests__/element-queries.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import {render} from '../'
+import '../../extend-expect'
```

Comment:
The problem with this is it means you have to run the build before you can run the tests (because on a fresh clone the `dist` directory doesn't work). It also means that any changes to `jest-extensions.js` wont be picked up by tests until the next build.

Instead, you'll basically need to duplicate the code.

Alternatively... You could put the code that's currently in `extend-expect.js` into `src/extend-expect.js` and then change the contents of: `extend-expect.js` to be simply: `require('./dist/extend-expect')`. Then you could update this import to simply: `'../extend-expect'` :+1: